src/js/common/schemaform/actions.js

```diff
@@ -6,6 +6,10 @@ export const SET_DATA = 'SET_DATA';
 export const SET_PRIVACY_AGREEMENT = 'SET_PRIVACY_AGREEMENT';
 export const SET_SUBMISSION = 'SET_SUBMISSION';
 export const SET_SUBMITTED = 'SET_SUBMITTED';
+export const SET_SAVED = 'SET_SAVED';
```

Comment:
This is my pass at the naming:

`saveInProgressForm`
`fetchInProgressForm`
`setFetchInProgressStatus`
`setSaveInProgressStatus`
`loadInProgressFormData`